test/solution.jl

```diff
@@ -146,7 +146,7 @@ using ODEProblemLibrary: prob_ode_lotkavolterra
                                 for i in 1:n_samples
                             )
                             ) / (m * n * o)
-                        @test_broken low <= percent_in_interval <= high
+                        @test low <= percent_in_interval <= high
```

Comment:
Tests that depend on RNG are probably a bad idea anyways so let's just skip this test for now
```suggestion
                        @test_skip low <= percent_in_interval <= high
```